### DISCUSSION TITLE
How to launch the linux version?

### DISCUSSION THREAD

**Discussion by KingAlejandro** [Kron4ek/minecraft-vortex-launcher#24] [Q&A] [answered]
I'm a bit of a noob with Linux and can't seem to work out how to launch the program 😳

**Comment by Kron4ek** [comment, marked as answer]
Ok, i'll try to explain. Open a terminal emulator and execute this command (press Enter after you enter it):

```
chmod +x /home/username/Downloads/VLauncher_1.1.14_x64_Linux
```

Instead of `/home/username/Downloads` you need to write the actual path to the directory where the VLauncher binary resides.

After that you can probably run the launcher by simply double-clicking it with your mouse. But if double-click doesn't work, you can run it through a terminal emulator. Enter the below command and press Enter:

```
/home/username/Downloads/VLauncher_1.1.14_x64_Linux
```

Again, you need to replace `/home/username/Downloads` with the actual path.

**Reply by KingAlejandro** [reply]
Thank you so much! Of course, you have to make it executable...

**Comment by ghost** [comment]
what then need do for launch 1.2.2 version?
i'm stuck on error:

Unrecognized VM option 'UseConcMarkSweepGC'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
openjdk version "17.0.4.1" 2022-08-12
OpenJDK Runtime Environment (Red_Hat-17.0.4.1.1-3.fc37) (build 17.0.4.1+1)
OpenJDK 64-Bit Server VM (Red_Hat-17.0.4.1.1-3.fc37)

install another version?

**Reply by Kron4ek** [reply]
@azr13ll0 Your Java version is too new for 1.12.2, you need to use Java 8.
